get.sh

```diff
@@ -587,6 +587,7 @@ testJavaVersion()
 	if [ "$TEST_JDK_HOME" = "" ]; then
 		TEST_JDK_HOME=$SDKDIR/openjdkbinary/j2sdk-image
 	fi
+	echo "TEST_JDK_HOME=${TEST_JDK_HOME}" > ${TESTDIR}/job.properties
```

Comment:
We should add this after line 602

Reply:
Changed line to be after 602. 
I'd like to ask, the `TEST_JDK_HOME` shall be the same as if we put it on #L590?

Reply:
It is better to set it once we verify it

Reply:
Thanks Lan :)